release/release_utils.py

```diff
@@ -526,22 +540,12 @@ def purge(repo, ignored_files=False):
             cmd = 'hg -R %s purge' % repo
     execute(cmd)
 
-def clean_repo(repo, prompt):
-    if maybe_prompt_yn('Remove all modified files, untracked files and outgoing commits from %s ?' % repo, prompt):
-        ensure_group_access(repo)
-        if git_bare_repo_exists_at_path(repo):
-            return
-        revert(repo)
-        strip(repo)
-        purge(repo, all)
-        revert(repo) # avoids the issue of purge deleting ignored files we want to get back
-    print ''
-
-def clean_repos(repos, prompt):
-    if maybe_prompt_yn('Remove all modified files, untracked files and outgoing commits from:\n%s ?' % '\n'.join(repos), prompt):
-        for repo in repos:
-            if repo_exists(repo):
-                clean_repo(repo, False)
+def clean_repo(repo):
```

Comment:
Over 100 documentation strings are missing across the release scripts. I'm working on adding them (I think it's a good idea to do this before the next release while these scripts are fresh in my head), but I think the Lock Checker tasks have a higher priority so I plan to set this aside for right now.